install_config/install/advanced_install.adoc

```diff
@@ -402,6 +402,100 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-pre-install-checks]]
+=== Configuring Cluster Pre-install Checks
+
+Pre-install checks are a set of diagnostic tasks that run as part of the
```

Comment:
I think "are" would read better in this case. Replacing "Pre-install checks" with the pronoun "they", for example, clears this up for me